lib/Interpreter/Interpreter.cpp

```diff
@@ -349,9 +349,11 @@ namespace cling {
       Strm << Linkage << " int __cxa_atexit(void (*f) (void*), void*, void*);\n";
 
       // C atexit, std::atexit
+#if !defined(LLVM_ON_WIN32) || !defined(_M_CEE_PURE)
```

Comment:
RuntimeUniverse.h should remain [somewhat] optional in my opinion. Putting it there would be ugly and require the file always be included. If RuntimeUniverse.h is missing/screwy that would mean destructors would never be called, as well as any usage of atexit crashing on shutdown.